packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -70,7 +60,7 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   }, []);
 
   return (
-    <EditorContainer bg="black" as="main" isIFrame={isIFrame}>
+    <EditorContainer bg="black" height="25rem" width="43rem" {...rest}>
```

Comment:
@borisonr instead of passing in these styles props individually, what if this component just accepts a single optional `className` prop with the styles? that way it's more explicit that it accepts style overrides

Reply:
(if we go with the single className objext, we can also default the styles there)